app/serializers/grade_serializer_v2.rb

```diff
@@ -1,6 +1,5 @@
 class GradeSerializerV2 < ActiveModel::Serializer
```

Comment:
This should now be the regular `GradeSerializer` and the old one should be deleted